pkg/serverless/protocol.go

```diff
@@ -82,6 +82,8 @@ type Daemon struct {
 	InvcWg *sync.WaitGroup
 
 	extraTags []string
+
+	finishInvocationOnce sync.Once
```

Comment:
I would add a comment for this

Reply:
i.e. it should be re-set before each invocation

Reply:
✅